torchgeo/models/resnet.py

```diff
@@ -67,10 +61,7 @@ class ResNet18_Weights(WeightsEnum):  # type: ignore[misc]
     )
 
     SENTINEL2_RGB_SECO = Weights(
```

Comment:
This weight has a problem but I can't figure out what it is:
```console
$ pytest -m slow tests/models/test_resnet.py::TestResNet50::test_resnet_download[ResNet50_Weights.SENTINEL2_RGB_SECO]
...
_pickle.UnpicklingError: invalid load key, '<'.
```
The error doesn't occur with the previous weight file, and both the old and new weight files have the same checksum. We'll need to figure out what's going on here before this PR can be merged.